### ISSUE TITLE
Make language IDs compliant with new identifier guidelines

### ISSUE THREAD

**Issue by fr1zle** [fr1zle/vscode-elixir#113]
Docs: https://code.visualstudio.com/docs/languages/identifiers#_new-identifier-guidelines

This is a breaking change as it may have an influence on other extensions relying on the current ids. The impact will have to be investigated.

Primer: https://github.com/vscode-icons/vscode-icons/issues/1414

**Comment by JimiC** [comment]
Just a friendly advice. Since this is a breaking change if you are following [Semantic Versioning](https://semver.org/) the version number should be bumped to `2.0.0`.

**Comment by fr1zle** [comment]
Yup, probably the smart thing to do. Will still have to check if there is a way for me to find out if any extensions are depending on the identifiers.

**Comment by JimiC** [comment]
From what I know, only file icon extensions and the client of the said extension, if that extension has a server service, are depending on the `language ids`. You may probably want to give a heads up to other file icon extension author about this change.

**Comment by ahmetcetin** [comment]
@fr1zle when do you plan to merge this pr? I just those changes manually in my installation files, it worked, but would be nice to have it in distro.
